### PR TITLE
Docsp 9559

### PR DESCRIPTION
**Pull request by terakilobyte**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-9559

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/353fa48/java/docsworker-xlarge/DOCSP-9559/fundamentals/crud/read-operations/project/
### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/project.txt [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+   collection.find(Filters.empty())
+       .projection(Projections.fields(Projections.include("name")))
+       .forEach(doc -> System.out.println(doc));
+
+
+The projection document specifies that the read operation result should
+*include* the ``name`` field of each returned document. As a result, this
+projection implicitly excludes the ``qty`` and ``rating`` fields. Chaining
+this projection to ``find()`` with an empty query document and no sort
+document yields the following results:
```
Comment (kyuan-mongodb) [inline source/fundamentals/crud/read-operations/project.txt:80]: Question: Why is "no sort document" being mentioned when this page isn't about sorts?

**Review by kyuan-mongodb — CHANGES_REQUESTED** [review]
Looking great overall. Just some suggestions I think can help improve the page.

File: source/fundamentals/crud/read-operations/project.txt (hunk 1 above)
Comment (terakilobyte) [reply source/fundamentals/crud/read-operations/project.txt:80]: Good question. I'll remove it!

**Review by kyuan-mongodb — APPROVED** [review]
LGTM!

**Comment by terakilobyte** [comment]
@rozza and @jyemin ready for your review. Very few driver specific things.

File: source/fundamentals/crud/read-operations/project.txt [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  including all unspecified fields.
+
+These two methods of projection are mutually exclusive: if you
+explicitly include fields, you cannot explicitly exclude fields, and
+vice versa.
+
+.. important::
+
+   The ``_id`` field is **not** subject to these mechanics. You must
+   explicitly exlude the ``_id`` field if you do not want it returned.
```
Comment (jyemin) [inline source/fundamentals/crud/read-operations/project.txt:44]: exlude -> exclude

File: source/fundamentals/crud/read-operations/project.txt [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+     { "_id": 3, "name": "oranges", "qty": 6, "rating": 2 },
+     { "_id": 4, "name": "avocados", "qty": 3, "rating": 5 },
+
+In the following query, pass the projection to return the ``name``
+field of each document:
+
+.. code-block:: java
+   :emphasize-lines: 3
+
+   // return all documents with only* the name field
```
Comment (jyemin) [inline source/fundamentals/crud/read-operations/project.txt:67]: Did you mean `*only*`?

File: source/fundamentals/crud/read-operations/project.txt [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+   { "_id": 1, "name": "apples" }
+   { "_id": 2, "name": "bananas" }
+   { "_id": 3, "name": "oranges" }
+   { "_id": 4, "name": "avocados" }
+
+Despite the fact that this projection only explicitly included the
+``name`` field, the query returned the ``_id`` field as well.
+
+The ``_id`` field is a special case: it is always included in every query
+return unless explicitly excluded. That's because the ``_id`` field is a
```
Comment (jyemin) [inline source/fundamentals/crud/read-operations/project.txt:89]: return -> result ?

**Review by jyemin — APPROVED** [review]
Just a few nits.  No need to re-request review though, once addressed.

File: source/fundamentals/crud/read-operations/project.txt (hunk 3 above)
Comment (terakilobyte) [reply source/fundamentals/crud/read-operations/project.txt:67]: I think either formatting works. I've changed it to *only*

**Pull request merged**